src/lib/tzcode/private.h

```diff
@@ -173,11 +173,8 @@ time_t posix2time(time_t);
 #if (defined __GLIBC__ \
      || defined __FreeBSD__ || defined __NetBSD__ || defined __OpenBSD__ \
      || (defined __APPLE__ && defined __MACH__))
-# if !defined TM_GMTOFF && !defined NO_TM_GMTOFF
-#  define TM_GMTOFF tm_gmtoff
-# endif
 # if !defined TM_ZONE && !defined NO_TM_ZONE
-#  define TM_ZONE tm_zone
+#  //define TM_ZONE tm_zone
```

Comment:
dead code. commit "datetime: localtime.c switched to Tarantool struct tnt_tm"

Reply:
Yup - that should be cleaned, as unused.

Reply:
Or rather defined `NO_TM_ZONE`.

Reply:
Or rather defined `NO_TM_ZONE`.